Design.md

```diff
@@ -84,9 +84,12 @@ The release process integrates with Fedora's release milestones in the following
     - The `next` stream switches to weekly releases to closely track the GA content set.
 - Fedora General Availability
     - Fedora CoreOS re-orients its release schedule in the following way:
-        - Week 0 (GA release): triple release;`next` with latest Fedora N content
-        - Week 1: triple release; `testing` release promoted from previous `next`
-        - Week 3: triple release; `stable` release promoted from previous `testing`, now fully rebased to Fedora N. `next` and `testing` are now in sync.
+        - Week 0 (GA release): triple release:
+          - `testing` release promoted from previous `next`
+          - `next` release contains latest Fedora N content
+        - Week 2: triple release:
+          - `stable` release promoted from previous `testing`, now fully rebased to Fedora N
+          - `testing` and `next` are now in sync
 
```

Comment:
I wonder if somewhere in here we should describe our behavior we've been doing where we fast-track changes that have landed in N-1 before N. i.e. our work to ensure "no downgrades" when upgrading FCOS.

Reply:
I think that probably belongs more in [the rebase checklist](https://github.com/coreos/fedora-coreos-tracker/blob/main/.github/ISSUE_TEMPLATE/rebase.md)?
Which actually, we should probably link from here too.